app/controllers/api/v3/appointments_controller.rb

```diff
@@ -15,10 +13,6 @@ def metadata
 
   private
 
-  def facility_group_records
```

Comment:
We've removed these individual calls to `syncable_to_region` in favour of a `region_records` that behaves the exact same way [everywhere](https://github.com/simpledotorg/simple-server/pull/1452/files#diff-40d0adce51092a7a607ada431136b5c8dc5447e754449277afd39d5a42c5a826R5). Exceptions are added as necessary to a couple of controllers.

This keeps the proximity of `region_records` with the methods that actually make use of it. So now,

* `region_records`
* `current_facility_records`
* `other_facility_records`

All, fly together, rather than cut awkwardly in different places among different controllers.